src/devices/Display/Font14.cs

```diff
@@ -0,0 +1,400 @@
+
+        /// <summary>
+        /// Digit 3
+        /// </summary>
+        Digit_3 = Segment14.Top | Segment14.TopRight | Segment14.BottomRight | Segment14.Bottom | Segment14.Middle,
+
+        /// <summary>
+        /// Digit 4
+        /// </summary>
+        Digit_4 = Segment14.TopRight | Segment14.BottomRight | Segment14.TopLeft | Segment14.Middle,
+
+        /// <summary>
+        /// Digit 5
+        /// </summary>
+        Digit_5 = Segment14.Top | Segment14.BottomRight | Segment14.Bottom | Segment14.TopLeft | Segment14.Middle,
+
+        /// <summary>
+        /// Digit 6
+        /// </summary>
+        Digit_6 = Segment14.Top | Segment14.BottomRight | Segment14.Bottom | Segment14.BottomLeft | Segment14.TopLeft | Segment14.Middle,
+
+        /// <summary>
+        /// Digit 7
+        /// </summary>
+        Digit_7 = Segment14.Top | Segment14.TopRight | Segment14.BottomRight,
+
+        /// <summary>
+        /// Digit 8
+        /// </summary>
+        Digit_8 = Segment14.Top | Segment14.TopRight | Segment14.BottomRight | Segment14.Bottom | Segment14.BottomLeft | Segment14.TopLeft | Segment14.Middle,
+
+        /// <summary>
+        /// Digit 9
+        /// </summary>
+        Digit_9 = Segment14.Top | Segment14.TopRight | Segment14.BottomRight | Segment14.Bottom | Segment14.TopLeft | Segment14.Middle,
+
+        /// <summary>
+        /// Hexadecimal digit A (upper case letter A)
+        /// </summary>
+        Digit_A = Letter_A,
+
+        /// <summary>
+        /// Hexadecimal digit B (letter b)
+        /// </summary>
+        Digit_B = Letter_b,
+
+        /// <summary>
+        /// Hexadecimal digit C (upper case letter C)
+        /// </summary>
+        Digit_C = Letter_C,
+
+        /// <summary>
+        /// Hexadecimal digit D (letter d)
+        /// </summary>
+        Digit_D = Letter_d,
+
+        /// <summary>
+        /// Hexadecimal digit E (upper case letter E)
+        /// </summary>
+        Digit_E = Letter_E,
+
+        /// <summary>
+        /// Hexadecimal digit F (upper case letter F)
+        /// </summary>
+        Digit_F = Letter_F,
+
+        /// <summary>
+        /// Lower case letter a
+        /// </summary>
+        Letter_a = Segment14.Top | Segment14.TopRight | Segment14.BottomRight | Segment14.Bottom | Segment14.BottomLeft | Segment14.Middle,
+
+        /// <summary>
+        /// Upper case letter A
+        /// </summary>
+        Letter_A = Segment14.Top | Segment14.TopRight | Segment14.BottomRight | Segment14.BottomLeft | Segment14.TopLeft | Segment14.Middle,
+
+        /// <summary>
+        /// Lower case letter b
+        /// </summary>
+        Letter_b = Segment14.BottomRight | Segment14.Bottom | Segment14.BottomLeft | Segment14.TopLeft | Segment14.Middle,
+
+        /// <summary>
+        /// Upper case letter B (same as digit 8)
+        /// </summary>
+        Letter_B = Digit_8,
```

Comment:
I think there are better representations for the letter "B" than the number "8". See e.g. [this image](https://commons.wikimedia.org/wiki/File:14-segment_display_B.svg). 

[Here](https://commons.wikimedia.org/wiki/File:14-segment_ASCII.svg) is also a full font for 14-segment displays. While at least some of the lower case letters look quite ugly, it could be used as a reference. (and note that this full font even has a different representation of "B" than the image above - I personally prefer the one which uses the center vertical bar.

Reply:
Love it, will go through these.

Reply:
Font updated in latest commit